packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 * Added
   * Migrate docs from terra-navigation-layout
   * Migrate docs from terra-menu
+  * Migrate docs from terra-theme-provider
```

Comment:
Not a big deal but some migration guides are "Added" whereas others are "Changed". Would be nice to be consistent. I think they should be "Added" since they are new additions to terra-framwork-docs.